packages/sdk/src/utils/chain-constants.ts

```diff
@@ -218,6 +219,22 @@ export const CONTRACT_ADDRESSES: {
     },
     l2: DEFAULT_L2_CONTRACT_ADDRESSES,
   },
+  [L2ChainID.BASE_MAINNET]: {
+    l1: {
+      AddressManager: '0x8EfB6B5c4767B09Dc9AA6Af4eAA89F749522BaE2' as const,
+      L1CrossDomainMessenger:
+        '0x866E82a600A1414e583f7F13623F1aC5d58b0Afa' as const,
+      L1StandardBridge: '0x3154Cf16ccdb4C6d922629664174b904d80F2C35' as const,
+      StateCommitmentChain:
+        '0x0000000000000000000000000000000000000000' as const,
+      CanonicalTransactionChain:
+        '0x0000000000000000000000000000000000000000' as const,
+      BondManager: '0x0000000000000000000000000000000000000000' as const,
+      OptimismPortal: '0x49048044D57e1C92A77f79988d21Fa8fAF74E97e' as const,
+      L2OutputOracle: '0x56315b90c40730925ec5485cf004d835058518A0' as const,
+    },
+    l2: DEFAULT_L2_CONTRACT_ADDRESSES,
```

Comment:
This included OVM_ETH and OVM_DeployerWhitelist which we probably don't need? But following what others have done

Reply:
Yup best to just keep it consistent for now we will delete these at a later time